scripts/issue/23/scrape_tokens.py

```diff
@@ -0,0 +1,39 @@
+from brownie import config
+import requests
+import pandas as pd
+
+def main():
+    """
+    Get all ERC-20 tokens and their balances in a csv for a given address with ethplorer api
+    
+    `scrape` can be called directly from command line with an address:
+    brownie run scrape_tokens.py scrape <address>
+    
+    edit `wallet_address` if you are running from brownie console
+    """
+    
+    wallet_address = ''
+    scrape(wallet_address)
+
+def scrape(address):
+    with requests.session() as session:
+        ethplorer_key = config['keys']['ETHPLORER_API_KEY']
+        api_key = ethplorer_key if ethplorer_key else 'freekey'
+        res = session.get(
+            f'https://api.ethplorer.io/getAddressInfo/{address}?showETHTotals=false', 
+            params={"apiKey": api_key}
+            )
+        assert res.status_code == 200
+
+    token_data = {'token_address': [], 'value': []}
+    for token in res.json()['tokens']:
+        try:
+            if int(token['tokenInfo']['decimals']) > 0:
+                token_data['token_address'].append(token['tokenInfo']['address'])
+                token_data['value'].append(token['rawBalance'])
+        except KeyError:
+            # Non-ERC20 token
+            pass
+        
+    pd.DataFrame(token_data).to_csv(f'scripts/issue/23/{address}_token_balances.csv')
```

Comment:
i probably should have specified; the gnosis airdrop csv has the following (strict) format:
```
token_type,token_address,receiver,value,id
```
> Transfer files are expected to be in CSV format with the following required columns:
>
>    token_type: The type of token that is being transferred. One of erc20,nft or native. NFT Tokens can be either ERC721 or ERC1155.
>    token_address: Ethereum address of ERC20 token to be transferred. This has to be left blank for native (ETH) transfers.
>    receiver: Ethereum address of transfer receiver.
>    value: the amount of token to be transferred. This can be left blank for erc721 transfers.
>    id: The id of the collectible token (erc721 or erc1155) to transfer. This can be left blank for native and erc20 transfers.
>
> Important: The CSV file has to use "," as a separator and the header row always has to be provided as the first row and include the described column names.

have a look at 6d6a95fe981f880391dadd68e28de6dce9e91309

my idea was then to leave `receiver` empty (or optional parameter for the func?).

simple use case is sending all dust on techops to treasury ops for processing, all in one go.